src/shared/components/msktable/MSKTable.tsx

```diff
@@ -436,27 +437,37 @@ export default class MSKTable<T> extends React.Component<MSKTableProps<T>, {}> {
         />
     }
 
-    render() {
+    buildTextBetweenTheButtons(): string {
```

Comment:
Look at `MSKTableStore.setProps`

When we had talked about this, Aaron preferred to keep all this logic in the store.